src/main/webapp/js/user-page-loader.js

```diff
@@ -142,7 +142,11 @@ function addPageButtons(){
       fetch("/numberOfMessages").then((response) => {
         return response.text();
       }).then((count) => {
-          var buttonCount=parseInt(count)/20;
+          int NumberOfMessages=20;
```

Comment:
In Javascript, we can't use int.

Reply:
Thanks for the help waqar. I keep forgetting that JS has different declarations